### ISSUE TITLE
pru-gcc vs. gcc-pru

### ISSUE THREAD

**Issue by MarkAYoder** [VedantParanjape/simpPRU#2]
The main README.md referecnes pru-gcc, but it should be gcc-pru.

**Comment by MarkAYoder** [comment]
Requirements need to include **cmake**

**Comment by MarkAYoder** [comment]
`docs/language-reference/language.md` has several places where ````python` and ` ``` `is used.  To format correctly it must be at the start of a line to format correctly .  i.e.:

```python

**Comment by MarkAYoder** [comment]
Help says **BBB**
```
simppru --help
Usage: simppru [OPTION...] FILE
Compiler for simpPRU, compiles simpPRU code down to PRU-C

      --device=<device_name> Select for which BeagleBoard to compile
                             (PocketBeagle, BBB, BBB Wireless, BB AI)
```
But the command is **bbb**

`simppru --device=bbb led_blink.sim`

**Comment by VedantParanjape** [comment]
> The main README.md referecnes pru-gcc, but it should be gcc-pru.

solved in f895831

**Comment by VedantParanjape** [comment]
> Requirements need to include **cmake**

solved in f895831

**Comment by VedantParanjape** [comment]
> Help says **BBB**
> 
> ```
> simppru --help
> Usage: simppru [OPTION...] FILE
> Compiler for simpPRU, compiles simpPRU code down to PRU-C
> 
>       --device=<device_name> Select for which BeagleBoard to compile
>                              (PocketBeagle, BBB, BBB Wireless, BB AI)
> ```
> 
> But the command is **bbb**
> 
> `simppru --device=bbb led_blink.sim`

solved in f895831

**Comment by VedantParanjape** [comment]
> `docs/language-reference/language.md` has several places where ````python`and` ``` `is used. To format correctly it must be at the start of a line to format correctly . i.e.:

Hey, could you elaborate a bit, I am unable to understand the problem here.

Thanks.

**Comment by MarkAYoder** [comment]
Take a look at the formatted /language.md.   Scroll down a ways and you'll see ```python in the text after Integer
![image](https://user-images.githubusercontent.com/660054/89306562-0b0ded80-d63e-11ea-938e-fa2b1d927abd.png)

The raw file shows the ```python is indented.
![image](https://user-images.githubusercontent.com/660054/89307037-9be4c900-d63e-11ea-9abc-acd75de28c6e.png)

If you remove the spaces before the ```python, it will format correctly.

**Comment by VedantParanjape** [comment]
Hey, 
Actually it is not meant to be read in .md format. These files are for the mkdocs generator. Which does not use vanilla Markdown. Unfortunately, github can't process it the way mkdocs will.

For docs generated using these markdown please take a look at: https://simppru.readthedocs.io/en/latest/
For the functions language reference: https://simppru.readthedocs.io/en/latest/language-reference/language/#functions

> If you remove the spaces before the ```python, it will format correctly.
If I remove spaces, it will break mkdocs generated output.

Thanks and Regards,
Vedant Paranjape

**Comment by MarkAYoder** [comment]
@VedantParanjape if you run simppru and gcc-pru isn't installed, the error meswsage is pru-gcc: not found.  It should be gcc-pru not found.

**Comment by VedantParanjape** [comment]
@MarkAYoder Thanks for the report, I have pushed the fix.

**Comment by MarkAYoder** [comment]
There's a reference to pru-gcc on the main page of https://simppru.readthedocs.io/en/latest/

**Comment by archisman-dey** [comment]
Fixed in #23
